addon/components/pix-input.hbs

```diff
@@ -16,21 +16,24 @@
   <div class="pix-input__container">
     <input
       id={{this.id}}
-      class={{if @errorMessage "pix-input__input--error"}}
+      class="pix-input__input--default {{this.validationStatusClassName}}"
       value={{@value}}
       aria-label={{this.ariaLabel}}
       aria-required="{{if @requiredLabel true false}}"
       required={{if @requiredLabel true false}}
-      aria-describedby={{if @errorMessage "text-input-error"}}
+      aria-describedby={{if (eq @validationStatus "error") "text-input-error"}}
```

Comment:
comment fonctionne eq ? comme une boucle for ?

Reply:
c'est un equal :) 

ça donnerait `@validationStatus === "error"`

Reply:
Ah bah oui 🤦🏻‍♀️, j'arrivais au même résultat mais avec cette lecture c'est plus simple 😅